templates/common/infra/bicep/core/database/cosmos/mongo/cosmos-mongo-cluster.bicep

```diff
@@ -0,0 +1,62 @@
+metadata description = 'Azure Cosmos DB MongoDB vCore cluster'
+@maxLength(40)
+param name string
+param location string = resourceGroup().location
+param tags object = {}
+
+param administratorLogin string

```

Comment:
Ideally, we would have some description for each parameter

Reply:
I removed the description to follow the biceps for other databases. 😂
I will add it again.